docs/index.md

```diff
@@ -155,7 +155,7 @@ Mocha allows you to use any assertion library you wish.  In the above example, w
 
 ## Asynchronous Code
 
-Testing asynchronous code with Mocha could not be simpler! Simply invoke the callback when your test is complete. By adding a callback (usually named `done`) to `it()`, Mocha will know that it should wait for this function to be called to complete the test.
+Testing asynchronous code with Mocha could not be simpler! Simply invoke the callback when your test is complete. By adding a callback (usually named `done`) to `it()`, Mocha will know that it should wait for this function to be called to complete the test. This callback accepts both an `Error` (or subclassed) instance, or `undefined`, any non-falsy will cause a failed test.
```

Comment:
Sorry, let's just eliminate `undefined` altogether here.  it's also a bit of a run-on sentence.  how about:

```markdown
This callback accepts both an `Error` instance (or subclass thereof) *or* a falsy value; anything else will cause a failed test.
```